pkg/storaged/nfs-panel.jsx

```diff
@@ -50,17 +50,11 @@ export class NFSPanel extends React.Component {
                     <td>{ server + " " + remote_dir }</td>
                     <td>{ entry.fields[1] }</td>
                     <td>
-                        { entry.mounted
+                        { entry.mounted && fsys_size
                             ? <StorageUsageBar stats={fsys_size} critical={0.95} />
                             : _("Not mounted")
```

Comment:
This will say "Not mounted" when we don't have size statistics, which is not correct.  With the old code `StorageUsageBar` would handle `stats == null` internally, and I think we should keep that.

Reply:
Fixed.